modules/peer-observer/default.nix

```
@@ -140,7 +140,7 @@ in {
 
     };
   };
-  config = mkIf (cfg.extractors.ebpf.enable || cfg.metrics.enable || cfg.addrConnectivity.enable || cfg.websocket.enable) {
+  config = mkIf (cfg.extractors.ebpf.enable || cfg.tools.metrics.enable || cfg.tools.addrConnectivity.enable || cfg.tools.websocket.enable) {
```

nice, thanks. I just saw that the rpc extractor is missing here too..


```suggestion
  config = mkIf (cfg.extractors.ebpf.enable || cfg.extractors.rpc.enable || cfg.tools.metrics.enable || cfg.tools.addrConnectivity.enable || cfg.tools.websocket.enable) {
```

I wonder if there's a better way of doing this

---

it has the benefit of being explicit :)